data/transactions/logic/assembler.go

```diff
@@ -825,10 +837,20 @@ func asmIntCBlock(ops *OpStream, spec *OpSpec, args []string) error {
 		}
 		l = binary.PutUvarint(scratch[:], cu)
 		ops.pending.Write(scratch[:l])
-		if !ops.known.deadcode {
+		if !ops.known.deadcode { // this conditional doesn't seem necessary
```

Comment:
It allows `intcblock` in deadcode to exist without mucking up the assemblers belief about what is in the cblock.